src/runtime/providers/prismic.ts

```diff
@@ -0,0 +1,24 @@
+import type { ProviderGetImage } from 'src'
+import { joinURL, parseQuery, parseURL, stringifyQuery } from 'ufo'
+import { operationsGenerator } from './imgix'
+
+const PRISMIC_IMGIX_BUCKET = 'https://images.prismic.io'
+
+// Prismic image bucket is left configurable in order to test on other environments
+export const getImage: ProviderGetImage = (
+  src,
+  { modifiers = {}, baseURL = PRISMIC_IMGIX_BUCKET } = {}
+) => {
+  const operations = operationsGenerator(modifiers)
```

Comment:
Perf: We `serialize` modifiers to `operations` and use `parseQuery` in return statement! (postponing since it is mostly a utility refactor for all providers with `operationsGenerator`)